src/js/reducers.js

```diff
@@ -447,38 +448,45 @@ function ui(state = {}, action) {
                 }
                 app.countDirection = action.updateMode
                 app.updateTime = new Date().getTime()
+                app.timerOffset = null
             }
             else if (action.updateMode === "COUNTDOWN") {
                 if (action.updateMode !== app.countDirection) {
                     app.endTime = action.endTime ? action.endTime : null
                 }
                 app.countDirection = action.updateMode
                 app.updateTime = new Date().getTime()
+                app.timerOffset = null
```

Comment:
Wouldn't it be better to set the value to 0(instead of null) and directly use `app.timerOffset` in `MediaTrackInfo_c.js` and `ProgressBar_c.js`